src/auth_manifests.yaml

```
@@ -0,0 +1,64 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app: {{ app_name }}
+  name: {{ app_name }}
```

I think for ClusterRoles we should add a bit more context to the naming since they're globally scoped (I realise that you can't atm deploy two kubeflows to the same k8s cloud, but maybe some day ... :)  ).  What about this?
```suggestion
  name: {{ namespace }}:{{ app_name }}
```

Could also do "-".  I see both ":" and "-" being used by different applications in `kubectl get clusterrolebinding`